app/handlers/karma.py

```diff
@@ -20,28 +22,36 @@
 
 @router.message(Command("top", prefix="!"), F.chat.type == "private")
 async def get_top_from_private(
-    message: types.Message, user: User, chat_repo: ChatRepo, user_repo: UserRepo
+    message: types.Message,
+    user: User,
+    chat_repo: ChatRepo,
+    user_repo: UserRepo,
 ):
-    parts = message.text.split(maxsplit=1)
-    if len(parts) == 1:
-        return await message.reply(
+    error_message = None
+    try:
+        chat = await get_chat(message.text.split(maxsplit=1), chat_repo)
```

Comment:
Any changes? I can commit this changes or just we need to cancel conversation and refactor this in another issue, because this review doesn't really fit this issue